suggested-companies/ch.json

```diff
@@ -0,0 +1,28 @@
+[
+	"ronorp",
+	"movu",
+	"immoscout24-ch",
+	"dlmoneypak",
+	"axa-ch",
+	"listglobally",
+	"tutti",
+	"tamedia-advertising",
+	"alle-immobilien",
+	"allekleinanzeigen",
+	"mitula",
+	"immowelt",
+	"urbanhome",
+	"wgzimmer",
+	"icasa",
+	"immoclick",
+	"flatfox",
+	"alacasa",
+	"betterhomes",
+	"engelvoelkers",
+	"eraimmobilien",
+	"goodliving",
+	"immofield",
+	"immosky",
+	"remax",
+	"nzz-domizil",
+]
```

Comment:
Thanks for adding suggested companies. I am not sure if these are the companies we are looking for, though. In 
#230, there's an explanation of the companies we are looking for here:

> With our suggested companies lists, we collect companies that have data on most people in their respected countries. Prominent examples for such companies include credit agencies and list brokers. We recommend sending GDPR access requests to all these companies.
This explicitly doesn't include companies that are relevant for many but not all users (like Google, Facebook, Amazon etc.)

I also [listed some common categories of companies](https://github.com/datenanfragen/data/issues/230#issuecomment-538915509).

Correct me if I'm wrong but I don't think companies like Immowelt match these criteria. What do you think?

Reply:
hey @baltpeter, I wasn't quite sure if these companies would fit into the objective of datenanfragen. I created this list because there seems to be some kind of web of data exchange between many of the online real estate companies in Switzerland. This fact makes it hard to know where your information ends up, even if you only use one of the companies in question. I understand if you want to keep datenanfragen more towards list brokers - in that case, just reject my merge requests ;) Otherwise, I will fix the formatting.

Reply:
Sorry for the late reply.

We are definitely interested in adding the companies. I just wouldn't want to add them to the suggested companies list for the reasons I gave.

I can definitely see what you wanted to achieve here, though. Maybe a better medium for this would be a blog post? Otherwise, let's just add the companies. :)